vespa-http-client/src/main/java/com/yahoo/vespa/http/client/core/XmlFeedReader.java

```diff
@@ -23,21 +24,21 @@ private XmlFeedReader() {}
 
     public static void read(InputStream inputStream, FeedClient feedClient, AtomicInteger numSent) throws Exception {
 
-        SAXParserFactory parserFactor = SAXParserFactory.newInstance();
-        parserFactor.setValidating(false);
-        parserFactor.setNamespaceAware(false);
-        final SAXParser parser = parserFactor.newSAXParser();
+        SAXParserFactory parserFactory = SAXParserFactory.newInstance();
+        // XXE prevention:
+        parserFactory.setFeature("http://xml.org/sax/features/external-general-entities", false);
```

Comment:
Should more from https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Prevention_Cheat_Sheet#SAXReader be included as well?

Reply:
I believe this disables all, and we don't use a SAXReader here.